erpnext/accounts/doctype/pos_profile/pos_profile.py

```diff
@@ -52,14 +52,23 @@ def validate_duplicate_groups(self):
 		if len(customer_groups) != len(set(customer_groups)):
 			frappe.throw(_("Duplicate customer group found in the cutomer group table"), title = "Duplicate Customer Group")
 
-	def check_default_payment(self):
-		if self.payments:
-			default_mode_of_payment = [d.default for d in self.payments if d.default]
-			if not default_mode_of_payment:
-				frappe.throw(_("Set default mode of payment"))
-
-			if len(default_mode_of_payment) > 1:
-				frappe.throw(_("Multiple default mode of payment is not allowed"))
+	def validate_payment_methods(self):
+		if not self.payments:
+			frappe.throw(_("Payment methods are mandatory. Add atleast one payment methods."))
+
+		default_mode_of_payment = [d.default for d in self.payments if d.default]
+		if not default_mode_of_payment:
+			frappe.throw(_("Set default mode of payment"))
```

Comment:
I know you didn't modify this, but _please_ :stuck_out_tongue: 
```suggestion
			frappe.throw(_("Please select a default mode of payment"))
```